packages/ui/src/form/DynamicFormField/DynamicInput.tsx

```diff
@@ -38,9 +39,11 @@ const DynamicInput: FunctionComponent<DynamicInputProps> = ({
     value,
     isFloatingLabelEnabled,
     date,
+    inputDateFormat,
     ...rest
 }) => {
-    const { inputFormat } = date || { inputFormat: '' };
+    const inputFormat = inputDateFormat || date?.inputFormat || '';
```

Comment:
May I ask you to add some tests to cover this change?

Reply:
@serhii-tkachenko added test for covering this change. Thanks